Sming/Libraries/modbusino/samples/generic/README.rst

```diff
@@ -0,0 +1,16 @@
+Modbusino RTU generic sample
+============================
+
+.. highlight:: bash
+
+The sample code provides three element uint16_t array used for the modbus slave registers. These values are printed each second through UART1.
+mbSlaveLoopTimer checks for new data each 10 milliseconds (Only checks for new data. Serial communication is handled by hardware.)
```

Comment:
Needs updating to reflect actual operation i.e. register values are printed only when a valid WriteMultipleRegisters command is received, no timer.